inventory/README.md

```diff
@@ -0,0 +1,62 @@
+# Inventory component
+Inventory component for screencrash, with support for items and achievements. Custom made specifically for the show Apornas ö.
+Run component to handle backend calls and then add a web view with the `media`-component to visualize the result, or visit webpage at http://localhost:4218.
+Manual changes can be done through screencrash commands or via the update subpage (http://localhost:4218/update).
+
+Available items and achivements can be set up with configuration files. See more in [Required resources](#required-resources)
+
+## Requirements
+NodeJS v10.0+ . Make is nice but not required.
```

Comment:
I had to use v16, or I would get `ReferenceError: Event is not defined`.

Reply:
Updated requirement in readme to v15.0